modules/odc-rolling-back-helm-release.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-working-with-helm-charts-using-developer-perspective.adoc
+
+[id="odc-rolling-back-helm-release_{context}"]
+= Rolling back a Helm release
+
+If a release fails to go as per plans, you can rollback the Helm release to a previous version.
```

Comment:
s/If the release fails to go as per plan/If the release fails OR If the release fails because of any reason
OR
You can rollback a Helm release to the previous version if there are any failures.

Reply:
Reworded to: If a release fails, to make it minimal as per first suggestion.